minio/minio
# Fix FIFO enforcement to maintain a list

of oldest objects qualifying for deletion.
Use this list when bucket exceeds configured FIFO
quota to bring it within quota.

## Description


## Motivation and Context
Make FIFO quota enforcement more efficient by maintaining a list of objects eligible for deletion for buckets configured with FIFO quota.

## How to test this PR?
Set up a bucket with FIFO quota using `mc admin bucket quota` , and do `mc cp` to fill objects in bucket over configured quota. When crawler runs, oldest objects over quota should be cleaned up.

## Types of changes

## Checklist:
